libs/chiliui/chili/controls/object.lua

```diff
@@ -92,14 +102,19 @@ function Object:New(obj)
 	obj = obj or {}
 
 	--// check if the user made some lower-/uppercase failures
+	--[[
 	for i, v in pairs(obj) do
 		if (not self[i]) and (isstring(i)) then
 			local correctName = self.__lowerkeys[i:lower()]
 			if (correctName) and (obj[correctName] == nil) then
-				obj[correctName] = v
+				Spring.Echo("__lowerkeys failure for ", i, "instead of ", correctName, type (v), 'wrapping')
+				Spring.Utilities.TraceFullEcho(5, 30,30)
+				local f = function(...) Spring.Echo("__lowerkeys CALLED!", correctname) return v(...) end
+				obj[correctName] = f
 			end
 		end
 	end
+	]]--
```

Comment:
goog idea - guess it's not needed at all. Might be implemented for some migrations in the past.

Reply:
Yeah, the single usage of clone instead of Clone only threw me a single line of being specified, but isnt ever called anyway.